Dockerfile.amd64

```diff
@@ -61,7 +62,11 @@ COPY --from=bird /bird* /bin/
 COPY filesystem/ /
 
 # Add in confd config and templates
-COPY vendor/github.com/kelseyhightower/confd/etc/calico/ /etc/calico/
+RUN mkdir -p /etc/calico/confd/conf.d /etc/calico/confd/config /etc/calico/confd/templates
+
+RUN for i in bird.toml bird6.toml bird6_aggr.toml bird6_ipam.toml bird_aggr.toml bird_ipam.toml tunl-ip.toml; do curl -L https://raw.githubusercontent.com/projectcalico/confd/4e1d8b2f7914b646963971ec56edfa6190e05e25/etc/calico/confd/conf.d/$i -o /etc/calico/confd/conf.d/$i; done
```

Comment:
I don't think this will work with hardcoded commit-ids.

Instead, I think we need to introduce a make target that grabs these config files and stashes them locally.  Something like
```
mkdir -p filesystem/etc/calico/confd/conf.d
cp -r $(go list -m -f "{{.Dir}}" github.com/kelseyhightower/confd)/etc/calico/confd/conf.d filesystem/etc/calico/confd/conf.d
```

Then the `COPY filesystem/ /` command takes care of it.

Reply:
Fixed

Reply:
Fixed